modules/ztp-deploying-a-site.adoc

```diff
@@ -73,7 +73,7 @@ include::snippets/ztp_example-sno.yaml[]
 +
 [NOTE]
 ====
-For more information about BMC addressing, see the "Additional resources" section.
+For more information about BMC addressing, see the "Additional resources" section. `installConfigOverrides` and  `ignitionConfigOverride` are expanded in the example for ease of readability.  
```

Comment:
Up to you, but suggestion to avoid having lowercase literal text somewhat awkwardly start a sentence:
```suggestion
For more information about BMC addressing, see the "Additional resources" section. The `installConfigOverrides` and  `ignitionConfigOverride` fields are expanded in the example for ease of readability.  
```